src/styles/style.css

```diff
@@ -0,0 +1,269 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
Using `gap` with flexbox may not be supported in all browsers (especially older ones). If you need to support older browsers, consider using margins instead.